worker/include/RTC/RTCP/FeedbackPsFir.hpp

```diff
@@ -27,7 +27,12 @@ namespace RTC
 			{
 				uint32_t ssrc;
 				uint8_t sequenceNumber;
+#ifdef _MSC_VER
```

Comment:
Is this a compiler specific requirement? If not I'd suggest using `ifdef _WIN32` for Windows code. It's not impossible the compiler changes and grows native supoport for Clang on Windows.

Reply:
for better compatibility use `_WIN32`, already renamed.